lms/static/sass/discussion/views/_thread.scss

```diff
@@ -51,53 +57,13 @@ body.discussion, .discussion-module {
   }
 
   // comments layout
-  .comments {
-    @extend %ui-no-list;
-    border-radius: 0 0 3px 3px;
-    background: $gray-l6;
-    box-shadow: 0 1px 3px -1px $shadow inset;
-
-    > li {
-      border-top: 1px solid $gray-l4;
-      padding: ($baseline/2) $baseline;
-    }
-
-
-    blockquote {
-      background: $gray-l4;
-      border-radius: 3px;
-      padding: ($baseline/4) ($baseline/2);
-      font-size: 14px;
-    }
-
-    .comment-form {
-      @include clearfix();
-
-      .comment-form-input {
-        padding: ($baseline/4) ($baseline/2);
-        background-color: $white;
-        font-size: 14px;
-      }
-
-      .discussion-submit-comment {
-        @include blue-button;
-        float: left;
-        margin-top: 8px;
-      }
-
-      .wmd-input {
-        height: 40px;
-      }
-
-      .discussion-errors {
-        margin: 0;
-      }
-    }
+  .discussion-comment {
 
     .response-body {
       display: inline-block;
       margin-bottom: ($baseline/2);
-      width: flex-grid(10,12);
+      //width: calc((flex-grid(11,12)) - $comment-image-dimension);
```

Comment:
Is this needed still (since its commented out and you have a ``flex-grid``-based width)?